helm/prometheus/README.md

```diff
@@ -59,6 +59,7 @@ Parameter | Description | Default
 `retention` | How long to retain metrics | `24h`
 `routePrefix` | Prefix used to register routes, overriding externalUrl route | `/`
 `rules` | Prometheus alerting & recording rules | `{}`
+`rulesSelector` | Rules ConfigMap selector | `{}`
```

Comment:
what's the difference between `rules` and `rulesSelector`?

Reply:
nvm I saw it below, one is the actual rules and the other the labels of the configmap

Reply:
Correct :)